src/reference/types/enum/index.md

```diff
@@ -9,3 +9,228 @@ tags:
   - 'types'
   - 'enum'
 ---
+
+## Enum List
+
+- [Enum List](#enum-list)
```

Comment:
This table of content should be removed, since we have one on the right sidebar on documentation page.